updater.js

```diff
@@ -0,0 +1,67 @@
+const fetch = require("node-fetch")
+const path = require("path")
+const pug = require("pug")
+const https = require("https")
+const fs = require('fs').promises
+
+const url = "https://www.my.af.mil/gcss-af/USAF/api/quicklinks/cached?categorized=true&id=p7F11BC9F789430190178946F7E140005&siteId=sD22E5184744EFC540174558CFFA50008";
+
+
+const sleep = (time) => new Promise((res, rej) => {
+    try {
+        setTimeout(() => {
+            res()
+        }, time)
+    } catch (e) {
+        rej(e)
+    }
+});
+
+const getCurrentData = async () => {
+    const ca = await fs.readFile(path.join(__dirname, "CA.cert"))
+    let data
+    let attempt = 1;
```

Comment:
```suggestion
    let attempt = 0;
```

Reply:
I started at 1 so the logs will say "attempt 1" not "attempt 0" .  what should change is line [37](https://github.com/dadatuputi/aflinks/pull/3/files/dba1e0be89471d60bd25946f0d867536599af251#diff-755913a300f761834c95431b4203471d469913d679df70135523e5bc48593dc1R37) to be `>` not `>=`